README.md

```diff
@@ -1,10 +1,27 @@
-## Dsp-Ingest
+# Dsp-Ingest
 
+[![Github](https://img.shields.io/github/v/tag/dasch-swiss/dsp-ingest?include_prereleases&label=Github%20tag)](https://github.com/dasch-swiss/dsp-ingest)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 ![Build And Test](https://github.com/dasch-swiss/dsp-ingest/actions/workflows/ci.yml/badge.svg)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/3717de9ffb22413c98c23161a0242799)](https://app.codacy.com/gh/dasch-swiss/dsp-ingest/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
 [![Codacy Badge](https://app.codacy.com/project/badge/Coverage/3717de9ffb22413c98c23161a0242799)](https://app.codacy.com/gh/dasch-swiss/dsp-ingest/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_coverage)
 
-The dsp-ingest service is designed to streamline and optimize the process of ingesting and managing data within the DaSCH Service Platform.
-By automating the ingestion process, DSP-Ingest enables efficient data collection, transformation, transcoding, meta-data extraction, and storage, ensuring that the data is readily available for serving with Sipi.
-This service aims to enhance the overall performance and effectiveness of DSP applications by simplifying and accelerating the data ingestion workflow.
+The dsp-ingest service is designed to streamline and optimize the process of ingesting and managing data within the
+DaSCH Service Platform.
+By automating the ingestion process, DSP-Ingest enables efficient data collection, transformation, transcoding,
```

Comment:
Decide one version in the document: `dsp-infgest`, `DSP-Ingest` or maybe `DSP Ingest"?

Reply:
Good point, as discussed: `Dsp-Ingest` if written in humanreadable text.

Fixed in:
aa290828f716ec8091391ddbf9e66d18895c868a